src/Angor/Avalonia/Angor.Model.Implementation/Projects/Stage.cs

```diff
@@ -0,0 +1,11 @@
+using AngorApp.Model;
+
+namespace Angor.Model.Implementation.Projects;
+
+public class Stage : IStage
+{
+    public DateOnly ReleaseDate { get; set; }
+    public uint Amount { get; set; }
```

Comment:
money we represent as long normally

Reply:
unless this is the stage representation so it is a percent in which case it should be a decimal

Reply:
It's the stage representation and it's a number in sats internally, that can be converted to whatever we need in the UI (BTC or sats)